app/renderer/actions/Inspector.js

```diff
@@ -69,6 +70,9 @@ export const SET_ACTION_ARG = 'SET_ACTION_ARG';
 
 export const SET_CONTEXT = 'SET_CONTEXT';
 
+export const GET_APP_ID = 'GET_APP_ID';
```

Comment:
to follow our current model, this and others like it should be called `SET_APP_ID`, since that is the action you are dispatching--one that sets the app id in the state